ctru-sys/src/bin/docstring-to-rustdoc.rs

```diff
@@ -0,0 +1,50 @@
+use std::{env, fs, io};
+use std::path::Path;
+
+fn main() -> io::Result<()> {
+    let args: Vec<String> = env::args().collect();
+
+    let bindings_path = Path::new(args.get(1).expect("bindings.rs not provided in the args"));
+    let bindings_string: String = fs::read_to_string(bindings_path)?;
+
+    let parsed = bindings_string
+        .lines()
+        .map(|v| {
+            // Only modify lines with the following structure: `` #[doc ... ] ``
+            if v.trim_start().starts_with("#[doc") && v.trim_end().ends_with("]") {
+                v
+                    .replace("@brief", "")
+                    // Example: ``@param offset Offset of the RomFS...`` -> ``- offset Offset of the RomFS...``
+                    // Will improve in the future
+                    .replace("@param", "* ")
+                    .replace("@ref", "")
+                    .replace("@note", "")
+                    .replace("@return", "")
+                    .replace("@sa", "")
+                    .replace("< ", "")
+                    // Remove things like ``@param[out]``
+                    .replace("[out]", "")
+                    .replace("[in]", "")
+                    // Trim start of the Rustdoc: ``...= " ...`` -> ``...= "...``
+                    .replace("= \" ", "= \"")
+                    // Double pass because _most_ annotations are at the start
+                    .replace("= \" ", "= \"")
+            } else {
+                String::from(v)
+            }
+        })
+        .map(|v| {
+            // End-lines of bindings.rs are CRLF
+            v + "\r\n"
```

Comment:
Can we change this to just LF?

Reply:
The bindings.rs committed to master uses LF, so maybe you're on Windows and bindgen uses the default.